utils/documentation_tests.txt

```diff
@@ -31,6 +31,6 @@ src/transformers/models/blenderbot/modeling_blenderbot.py
 src/transformers/models/blenderbot_small/modeling_blenderbot_small.py
 src/transformers/models/plbart/modeling_plbart.py
 src/transformers/generation_utils.py
+src/transformers/models/resnet/modeling_resnet.py
```

Comment:
I kind get the idea. If I understand it correctly, here are my 2 cents:
- maybe move this line before `src/transformers/generation_utils.py`?
- maybe in a future PR, we can have a script to sort the lines by some order (say, alphabetic)

Reply:
Alphabetic sort is a very good idea!